ts/src/db_connection.ts

```diff
@@ -24,9 +28,22 @@ export function getDBConnection(): Connection {
 /**
  * Creates the DB connnection to use in an app
  */
-export async function initDBConnectionAsync(): Promise<void> {
+export async function initDBConnectionAsync(options?: ConnectionOptions): Promise<void> {
     if (!_.isUndefined(connectionIfExists)) {
         throw new Error('DB connection already exists');
     }
-    connectionIfExists = await createConnection();
+    let connOptions = options;
+    if (connOptions === undefined) {
+        connOptions = {
+            type: 'sqlite',
+            database: 'database.sqlite',
+            synchronize: true,
+            logging: true,
+            entities: [OrderEntity, TakerAssetFillAmountEntity, TransactionEntity],
+            cli: {
+                entitiesDir: './entities',
+            },
+        };
```

Comment:
I'd rather we put the default opts into it's own file and import it here. Something like `default_typeorm_configs.js`.